<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Проводник</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
background-color: rgb(34, 90, 229);
}
QListWidget
{
padding: 10px;
border-radius: 25px;
border : 1px solid black;
background : rgb(48, 36, 37);
color: white;
}
 QListView::item:selected
{
border : 1px solid black;
background : rgb(173, 153, 255);
}
 QListView::item
{
margin:3 px;
}
QPushButton {
background: rgb(13, 212, 46);
}
QLabel {
color: white;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_app_but">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>490</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(0, 0, 0);
border-radius: 30px;
color: white;
}
QPushButton:hover
{
background-color: rgb(255, 255, 255);
color: black;
}
QPushButton:pressed
{
background-color: rgb(255, 207, 16);
}</string>
    </property>
    <property name="text">
     <string>Запустить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Список аудиокниг</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Список радиостанций</string>
    </property>
   </widget>
   <widget class="QListWidget" name="books_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>291</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="radio_list">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>301</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="add_book_but">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px; 
border: 2px groove black;
padding-bottom: 5px;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_radio_but">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px; 
border: 2px groove black;
padding-bottom: 5px;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QListWidget" name="friend_list">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>301</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Onyx</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Список друзей</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_friend_but">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px; 
border: 2px groove black;
padding-bottom: 5px;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="presentation_but">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(0, 0, 0);
border-radius: 20px;
color: white;
}
QPushButton:hover
{
background-color: rgb(255, 255, 255);
color: black;
}
QPushButton:pressed
{
background-color: rgb(255, 207, 16);
}</string>
    </property>
    <property name="text">
     <string>ПР</string>
    </property>
   </widget>
   <zorder>friend_list</zorder>
   <zorder>start_app_but</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>books_list</zorder>
   <zorder>radio_list</zorder>
   <zorder>add_book_but</zorder>
   <zorder>add_radio_but</zorder>
   <zorder>label_3</zorder>
   <zorder>add_friend_but</zorder>
   <zorder>presentation_but</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
